output/cmf.eslint.txt

```diff
@@ -22,7 +22,10 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   177:51  error    There should be no spaces inside this paren  space-in-parens
 
 /home/travis/build/Talend/ui/packages/cmf/src/sagas/collection.js
-  9:1  error  Prefer default export  import/prefer-default-export
+  10:1  error  Prefer default export  import/prefer-default-export
 
-✖ 8 problems (3 errors, 5 warnings)
+/home/travis/build/Talend/ui/packages/cmf/src/UIRouter.js
+  27:78  error  'dispatch' is missing in props validation  react/prop-types
```

Comment:
can you clean the lint errors ?

Reply:
Oh, I haven't turned on eslint on my new computer, sorry for that